### PR TITLE
Microsoft.WindowsTerminal version 1.17.11461.0

### PR DESCRIPTION
**Pull request by DHowett**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108350)

### CODE REVIEW

**Comment by BrandonWanHuanSheng** [comment]
The pull request failed due to Microsoft.UI.Xaml.2.7. This pull request running in the sandbox.

**Comment by BrandonWanHuanSheng** [comment]
@stephengillie Can you add blocking issue with dependcies?  Otherwise, the pull request will closed automatically.

**Comment by floh96** [comment]
@stephengillie @DHowett winget 1.6 supports dependencies could this pr be updated?

**Comment by DHowett** [comment]
That's great news! Somebody will need to explain how to do that :smile:

**Comment by mdanish-kh** [comment]
IIRC, this depended on WinUI.Xaml version 2.8 correct? 👀

**Comment by mdanish-kh** [comment]
So, looks like it depends on -

- `Microsoft.VCLibs.140.00.UWPDesktop`
- `Microsoft.UI.Xaml.2.7`

Looks like we need to add VCLibs to the repository first. We already have WinUI Xaml 2.7 in the repo.

**Comment by DHowett** [comment]
Honestly, we might just want to skip this version. It is the last version that requires the `UWPDesktop` VCLibs package.

The version that is available online is--regrettably--insanely out of date. They will not release an updated version.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
```
2023-09-26 11:58:00.209 [CLI ] Completed installer selection.
2023-09-26 11:58:01.068 [CLI ] Installer hash verified
2023-09-26 11:58:01.178 [CORE] Starting RequestAddPackageAsync operation #0: https://github.com/microsoft/terminal/releases/download/v1.17.11461.0/Microsoft.WindowsTerminal_1.17.11461.0_8wekyb3d8bbwe.msixbundle
2023-09-26 11:58:01.183 [CORE] Begin waiting for operation #0
2023-09-26 11:58:01.183 [CORE] Begin blocking for operation #0
2023-09-26 11:58:03.021 [CORE] Deployment operation #0: Windows cannot install package Microsoft.WindowsTerminal_1.17.11461.0_x64__8wekyb3d8bbwe because this package depends on a framework that could not be found. Provide the framework "Microsoft.UI.Xaml.2.7" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 7.2208.15002.0, along with this package to install. The frameworks with name "Microsoft.UI.Xaml.2.7" currently installed are: {Microsoft.UI.Xaml.2.7_7.2109.13004.0_x64__8wekyb3d8bbwe}
2023-09-26 11:58:03.021 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!00007FFDF57B9A45: (caller: 00007FFDF5743515) Exception(1) tid(1998) 80073CF3 Package failed updates, dependency or conflict validation.


2023-09-26 11:58:03.022 [CORE] Starting RemovePackage operation #1: Microsoft.WindowsTerminal_3001.17.11461.0_neutral_~_8wekyb3d8bbwe
2023-09-26 11:58:03.033 [CORE] Begin waiting for operation #1
2023-09-26 11:58:03.033 [CORE] Begin blocking for operation #1
2023-09-26 11:58:04.414 [CORE] Deployment operation #1: Windows cannot remove Microsoft.WindowsTerminal_3001.17.11461.0_neutral_~_8wekyb3d8bbwe because the current user does not have that package installed. Use Get-AppxPackage to see the list of packages installed.
2023-09-26 11:58:04.414 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FFDF57B8724: (caller: 00007FFDF57B9DFC) Exception(2) tid(1998) 80073CF1 Package was not found.

    Msg:[Operation failed: Windows cannot remove Microsoft.WindowsTerminal_3001.17.11461.0_neutral_~_8wekyb3d8bbwe because the current user does not have that package installed. Use Get-AppxPackage to see the list of packages installed.] 

2023-09-26 11:58:04.414 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(126)\WindowsPackageManager.dll!00007FFDF593F371: (caller: 00007FFDF57BB123) LogHr(1) tid(1998) 80073CF1 Package was not found.

    Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FFDF57B8724: (caller: 00007FFDF57B9DFC) Exception(2) tid(1998) 80073CF1 Package was not found.

    Msg:[Operation failed: Windows cannot remove Microsoft.WindowsTerminal_3001.17.11461.0_neutral_~_8wekyb3d8bbwe because the current user does not have that package installed. Use Get-AppxPackage to see the list of packages installed.] 
] 

2023-09-26 11:58:04.419 [CLI ] MSIX installer failed: 2147958003
2023-09-26 11:58:04.421 [CLI ] Terminating context: 0x8a150104 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\InstallFlow.cpp:1bc
```

**Comment by mdanish-kh** [comment]
> Automatic Validation ended with:
> 
> ```
> 2023-09-26 11:58:00.209 [CLI ] Completed installer selection.
> 2023-09-26 11:58:01.068 [CLI ] Installer hash verified
> 2023-09-26 11:58:01.178 [CORE] Starting RequestAddPackageAsync operation #0: https://github.com/microsoft/terminal/releases/download/v1.17.11461.0/Microsoft.WindowsTerminal_1.17.11461.0_8wekyb3d8bbwe.msixbundle
> 2023-09-26 11:58:01.183 [CORE] Begin waiting for operation #0
> 2023-09-26 11:58:01.183 [CORE] Begin blocking for operation #0
> 2023-09-26 11:58:03.021 [CORE] Deployment operation #0: Windows cannot install package Microsoft.WindowsTerminal_1.17.11461.0_x64__8wekyb3d8bbwe because this package depends on a framework that could not be found. Provide the framework "Microsoft.UI.Xaml.2.7" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 7.2208.15002.0, along with this package to install. The frameworks with name "Microsoft.UI.Xaml.2.7" currently installed are: {Microsoft.UI.Xaml.2.7_7.2109.13004.0_x64__8wekyb3d8bbwe}
> 2023-09-26 11:58:03.021 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!00007FFDF57B9A45: (caller: 00007FFDF5743515) Exception(1) tid(1998) 80073CF3 Package failed updates, dependency or conflict validation.
> 
> 
> 2023-09-26 11:58:03.022 [CORE] Starting RemovePackage operation #1: Microsoft.WindowsTerminal_3001.17.11461.0_neutral_~_8wekyb3d8bbwe
> 2023-09-26 11:58:03.033 [CORE] Begin waiting for operation #1
> 2023-09-26 11:58:03.033 [CORE] Begin blocking for operation #1
> 2023-09-26 11:58:04.414 [CORE] Deployment operation #1: Windows cannot remove Microsoft.WindowsTerminal_3001.17.11461.0_neutral_~_8wekyb3d8bbwe because the current user does not have that package installed. Use Get-AppxPackage to see the list of packages installed.
> 2023-09-26 11:58:04.414 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FFDF57B8724: (caller: 00007FFDF57B9DFC) Exception(2) tid(1998) 80073CF1 Package was not found.
> 
>     Msg:[Operation failed: Windows cannot remove Microsoft.WindowsTerminal_3001.17.11461.0_neutral_~_8wekyb3d8bbwe because the current user does not have that package installed. Use Get-AppxPackage to see the list of packages installed.] 
> 
> 2023-09-26 11:58:04.414 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(126)\WindowsPackageManager.dll!00007FFDF593F371: (caller: 00007FFDF57BB123) LogHr(1) tid(1998) 80073CF1 Package was not found.
> 
>     Msg:[D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(54)\WindowsPackageManager.dll!00007FFDF57B8724: (caller: 00007FFDF57B9DFC) Exception(2) tid(1998) 80073CF1 Package was not found.
> 
>     Msg:[Operation failed: Windows cannot remove Microsoft.WindowsTerminal_3001.17.11461.0_neutral_~_8wekyb3d8bbwe because the current user does not have that package installed. Use Get-AppxPackage to see the list of packages installed.] 
> ] 
> 
> 2023-09-26 11:58:04.419 [CLI ] MSIX installer failed: 2147958003
> 2023-09-26 11:58:04.421 [CLI ] Terminating context: 0x8a150104 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\InstallFlow.cpp:1bc
> ```

Yep - I was going to suggest the dependency once https://github.com/microsoft/winget-pkgs/pull/120905 gets merged

**Comment by mdanish-kh** [comment]
> Honestly, we might just want to skip this version. It is the last version that requires the `UWPDesktop` VCLibs package.
> 
> The version that is available online is--regrettably--insanely out of date. They will not release an updated version.

Since you're the publisher - that's completely up to you 😄

File: manifests/m/Microsoft/WindowsTerminal/1.17.11461.0/Microsoft.WindowsTerminal.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+PackageVersion: 1.17.11461.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.19041.0
+Scope: user
+InstallModes:
+- silent
+Commands:
+- wt
+PackageFamilyName: Microsoft.WindowsTerminal_8wekyb3d8bbwe
```
Comment (stephengillie) [inline manifests/m/Microsoft/WindowsTerminal/1.17.11461.0/Microsoft.WindowsTerminal.installer.yaml:14]:
```suggestion
PackageFamilyName: Microsoft.WindowsTerminal_8wekyb3d8bbwe
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.UI.Xaml.2.7
```

Comment (mdanish-kh) [reply manifests/m/Microsoft/WindowsTerminal/1.17.11461.0/Microsoft.WindowsTerminal.installer.yaml:14]: In a sandbox environment, this package also requires VCLibs. We would have to do something like

```
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCLibs.Desktop.14
    MinimumVersion: 14.0.30704.0
  - PackageIdentifier: Microsoft.UI.Xaml.2.7
    MinimumVersion: 7.2208.15002.0
```

**Comment by DHowett** [comment]
As the maintainer, I'm content to let this one pass un-merged. We've superseded it, and it's available via other channels. :)

**Pull request closed without merging**